src/main/java/de/muenchen/isi/infrastructure/entity/stammdaten/VersorgungsquoteGruppenstaerke.java

```diff
@@ -0,0 +1,38 @@
+package de.muenchen.isi.infrastructure.entity.stammdaten;
+
+import de.muenchen.isi.infrastructure.entity.BaseEntity;
+import java.time.LocalDate;
+import javax.persistence.Column;
+import javax.persistence.Entity;
+import javax.persistence.Index;
+import javax.persistence.Table;
+import javax.persistence.UniqueConstraint;
+import lombok.Data;
+import lombok.EqualsAndHashCode;
+import lombok.ToString;
+
+@Entity
+@Table(
+    uniqueConstraints = { @UniqueConstraint(columnNames = { "gueltigAb", "bildungseinrichtungBezeichnung" }) },
+    indexes = { @Index(name = "versorgungsqoute_gruppen", columnList = "gueltigAb, bildungseinrichtungBezeichnung") }
+)
+@Data
+@ToString(callSuper = true)
+@EqualsAndHashCode(callSuper = true)
+public class VersorgungsquoteGruppenstaerke extends BaseEntity {
+
+    @Column(nullable = false)
+    private LocalDate gueltigAb;
+
+    @Column(nullable = false)
+    private String bildungseinrichtungBezeichnung;
```

Comment:
Wäre es hier nicht besser eine Enum zu definieren?

Leider kann der Enum für den Typ der Infrastruktureinrichtung nicht verwendet werden, da dieser keinen Hort kennt.

Hier ist ggf. ein eingener Enum zu definieren.

![image](https://github.com/it-at-m/isi-backend/assets/56300801/e411f1f3-4999-4614-a9b7-5ee97000ed40)